x-pack/plugins/cloud/public/components/chat/index.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* eslint-disable import/no-default-export */
```

Comment:
Looks like there is a way around this without having to add an eslint disable - https://github.com/reactjs/rfcs/pull/64#issuecomment-431383273 and https://github.com/facebook/react/issues/14603 have some context, but I think it'd be changing

```
export const LazyChat = React.lazy(() => import('./chat'));
```

to something like

```
export const LazyChat = React.lazy(async () => {
  const Components = await import('./chat');
  // Resolve to named export:
  return Components.Chat
});
```